shopify_function_macro/src/lib.rs

```diff
@@ -309,24 +312,16 @@ pub fn shopify_function_target(
         |module_name| Ident::new(module_name.value().as_str(), Span::mixed_site()),
     );
 
-    let query_path = args.query_path.expect("No value given for query_path");
-    let schema_path = args.schema_path.expect("No value given for schema_path");
+    let query_path = args.query_path.expect("No value given for query_path").value();
+    let schema_path = args.schema_path.expect("No value given for schema_path").value();
     let extern_enums = args.extern_enums.as_ref().map(extract_extern_enums);
-    let output_query_file_name = format!(".{}{}", &target_handle_string, OUTPUT_QUERY_FILE_NAME);
 
-    let input_struct = generate_struct(
-        "Input",
-        query_path.value().as_str(),
-        schema_path.value().as_str(),
+    let input_struct = generate_input_struct(
+        query_path.as_str(),
+        schema_path.as_str(),
         extern_enums.as_deref(),
     );
 
-    let output_struct = generate_struct(
-        "Output",
-        &output_query_file_name,
-        schema_path.value().as_str(),
-        extern_enums.as_deref(),
```

Comment:
@jbourassa Does the output struct need extern enums as well?

Reply:
Ideally, yes: if the output struct wants e.g. `CountryCode`, we don't want to bloat the binary size and have mismatching definitions for `CountryCode`.